src/Sulu/Component/Content/Types/Rlp/Mapper/PhpcrMapper.php

```diff
@@ -593,9 +593,7 @@ private function checkResourceLocator(NodeInterface $routeNode, $resourceLocator
                 // route already exists and referenced on contentNode
                 return true;
             } else {
-                throw new ResourceLocatorAlreadyExistsException(sprintf(
-                    'Resource locator "%s" already exists. Route node at path "%s"', $resourceLocator, $routeNode->getPath()
-                ));
+                throw new ResourceLocatorAlreadyExistsException($resourceLocator, $routeNode->getPath());
```

Comment:
Instead of adapting the standard constructor of the exception, I'd rather add a static factory method, such as `ResourceLocatorAlreadyExistsException::forResourceLocator()`. This is a generally useful practice that allows to add additional factory methods or subclassing the exception, while still being able to call the standard constructor and passing a custom message.